functors_task_1/student/src/impl.cpp

```diff
@@ -1,2 +1,44 @@
 #include "impl.h"
 
+#include <algorithm>
+#include <forward_list>
+#include <map>
+#include <utility>
+
+BeerBrand BeerOrganizer::operator()()
+{
+    if (m_nextBeer <= static_cast<BaseType>(BeerBrand::None)
```

Comment:
I think it is better to specify range explicitly with const variables: "to" and "from"

Reply:
You're right! Thanks!